test/e2e/manifest/manifest.go

```diff
@@ -87,6 +91,20 @@ func IngressFromManifest(fileName string) (*extensions.Ingress, error) {
 	return &ing, nil
 }
 
+// IngressToManifest generates a yaml file in the given path with the given ingress.
+// Returns the file name and error, if there was any.
```

Comment:
It doesn't return the filename, does it?

Reply:
removed